app/storage.tsx

```diff
@@ -61,6 +71,74 @@ class Storage {
     }, null);
   }
 
+  // Insert expansion pack into the database
+  parseExpansionPack(expansionData) {
+    this.db.transaction((tx) => {
+      tx.executeSql(
+        'INSERT INTO expansionPackTable (pack_name, description, wtw, organisation) VALUES (?,?,?,?)',
+        [
+          expansionData.meta.pack_name,
+          expansionData.meta.description,
+          expansionData.meta.w3w,
+          expansionData.meta.organisation
+        ]
+      );
+      tx.executeSql(
+        'SELECT id FROM expansionPackTable WHERE pack_name=?',
+        ['Boyd Orr'],
+        (_, results) => {
+          expansionData.UUIDs.forEach((value) => {
+            tx.executeSql(
+              'INSERT INTO uuidTable (id, nmw, name, description, website, expansionID) VALUES (?,?,?,?,?,?)',
+              [
+                value.code,
+                value.nmw,
+                value.name,
+                value.description,
+                value.website,
+                results.rows.item(0).id
+              ]
+            );
+          });
+        }
+      );
+    });
+  }
+
+  deleteExpansionPack(code: number) {
+    this.db.transaction((tx) => {
+      tx.executeSql('DELETE FROM uuidTable WHERE expansionID=?;', [code]);
+      tx.executeSql('DELETE FROM expansionPackTable WHERE id=?;', [code]);
+    });
+  }
+
+  // 'INSERT INTO versionRecord (version) VALUES (?)'
+  printExpansionPack() {
+    this.db.transaction((tx) => {
+      tx.executeSql('SELECT * FROM uuidTable', [], (_, results) => {
+        console.log(results.rows);
+      });
+      tx.executeSql('SELECT * FROM expansionPackTable', [], (_, results) => {
+        console.log(results.rows);
+      });
+    }, null);
+  }
+
+  getUUIDData(code: string, callback: Function) {
+    this.db.transaction((tx) => {
+      tx.executeSql(
+        'SELECT nmw FROM uuidTable WHERE nmw=?',
+        [code],
+        (_, results) => {
+          console.log(results, 'this is results ---');
+          callback(results.rows.item(0).nmw);
+        }
+      );
+      console.log('error error error');
+    });
+    // console.log("this is the code");
+    console.log(code);
+  }
```

Comment:
What should `getUUIDData()` do? My understanding is that it should lookup a UUID in the database and provide the resulting table row via the callback. That means we should be looking up by a `UUID` field in the table, not `nmw`, right? I think it may just be that the naming is a bit confusing, but the logic might be right.